content/guides/integration-guides/csi-integration/soda-csi-pnp/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: SODA Integration with Vendor CSI Drivers via Soda CSI Plug-N-Play
+menuTitle: "Soda CSI Plug-N-Play"
+description: "A guide for integrating SODA with Vendor CSI Driver via unified Plugin."
+weight: 10
+disableToc: false
+tags: ["integration guide", "kubernetes csi", "plug-n-play", "unified plugin"] 
+---
+
+SODA CSI Plug-N-Play is a mechanism by which users can use the heterogeneous storage vendors in a unified way, 
+Users need to define the requirements while creating SODA Profile, 
+Users need to use this profile ID while creating the PVC's in the Kubernetes, 
+the soda-csi-provisioner will automatically select the vendor csi-driver and help in 
+provisioning the Volumes for Pods.
+  
+## Prerequisite
+ - An installation of Kubernetes (V1.17+)  
+ - Soda installation (you can refer to quick start guide over [here](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/)), you need to change the ports of etcd in osdb.yaml while installation soda on an existing K8s env as the ports causes conflict with k8s etcd.
```

Comment:
nit: Soda -> SODA

while installation soda on an existing K8s env as the ports causes conflict with k8s etcd -> while **installing** **SODA** on an existing K8s **environment** as the ports **cause** conflict with **K8s** etcd

Reply:
updated